### DISCUSSION TITLE
Text looks dim and flushed out

### DISCUSSION THREAD

**Discussion by CodeWithShreyans** [ghostty-org/ghostty#3921] [Q&A] [answered]
Maybe I made some config error but I'm using the same font on Ghostty and Rio (my previous term) and the text looks very dim.

<img width="1440" alt="SCR-20241229-ulzb" src="https://github.com/user-attachments/assets/de4f3a35-1267-48bd-bffd-533bea0ea06e" />

**Comment by edrozenberg** [comment]
Check if this helps:
`bold-is-bright = true`

And try messing with the color palette colors, for ex. I use these atm:
```
palette = 04=#3940b2
palette = 06=#49a3b0
palette = 12=#0435f0
palette = 14=#69e2e4
```

Also try the web configurator, look at color option for "Foreground color":
https://ghostty.zerebos.com/settings/colors

And after making any changes you can see the generated config here:
https://ghostty.zerebos.com/app/import-export

**Comment by CodeWithShreyans** [comment]
this seems to be an issue with the `Github Dark` theme I'm using

**Comment by CodeWithShreyans** [comment, marked as answer]
Fixed it with `foreground = #ffffff`
